src/components/memorygame/MemoryGameFactList/index.jsx

```diff
@@ -1,31 +1,65 @@
 import React from "react";
 import "./index.css";
 import { List, Card } from "antd";
+import { useTranslation } from "react-i18next";
 
 /**
  * Upon matching cards, a fact related to matched cards appear on screen
  * This component creates the list that will hold all facts
  * "facts" is an array of objects
- * "title" is a string that is translated by i18n. It is used to describe what the list holds.
  * If the language is English "title" will be "Facts"
+ * "emptyFactListMessage" is a string that is translated by i18n.
+ * It is displayed when fact list is empty
+ * If the language is English "emptyFactListMessage" will be ""Match cards to reveal facts!"
  */
-const MemoryGameFactList = ({ facts, title }) => {
-  return (
-    <>
-      <h1 className="memoryGame__title">{title}</h1>
+const MemoryGameFactList = ({ facts }) => {
+  // facts example:
+  // {
+  //   cardKey: "alternateEnergy0",
+  //   description: "Alternatif Enerji",
+  //   img: {src: "/static/media/alternateEnergy.88cef00e.svg", imgKey: "alternateEnergy"},
+  //   isFlipped: false,
+  //   isMatched: false,
+  //   link: "https://loremIpsum.com",
+  //   phrase: "Lorem ipsum",
+  // }
+  const [t] = useTranslation();
+  const factsTitle = t("memoryGame.facts");
+  const emptyFactListMessage = t("memoryGame.emptyFactListMessage");
+  const factListTitle = (
+    <h1 className="memoryGameFactListContainer__title">{factsTitle}</h1>
+  );
+  let factListContent;
+  // facts is an empty array by default, it gets filled when user makes successful match
+  if (facts.length) {
```

Comment:
facts.length !== 0, be explicit, you may think it's shorter and fancy but it takes the reader a second to think about this

Reply:
PTAL